### DISCUSSION TITLE
Seen/Watched status

### DISCUSSION THREAD

**Discussion by fdw** [seerr-team/seerr#506] [Ideas]
I'm not sure if this fits within the scope of Jellyseerr, so feel free to just ignore 🙂

There's lots of movies and shows I've seen before, but that are not in my collection. I would love some way to track these somewhere and show them differently in Jellyseerr, so that I don't request them again.

**Comment by brianjmurrell** [comment]
Or movies/shows that are no longer in my collection because I have watched them and deleted them, but I would like to have them filtered out of my lists in Jellyseerr so that I am not constantly having to sift through those lists looking for things I have not yet watched.

One of the most annoying things about Netflix (and the like) is that their various lists show me lots of stuff I have already watched.  I would love nothing more than a "Only unwatched" filter/checkbox to remove items from showing in lists that I have already watched before.

**Comment by norbertjoni** [comment]
I just came here for this. Would be possible with Trakt.tv or tautulli integration

**Reply by fallenbagel** [reply]
We already have tautulli integration?
Since versions ago

**Reply by 0xSysR3ll** [reply]
Traktv API is a real mess

**Comment by Szymok** [comment]
Yess! It would be lovely to add some button next to Watchlist/Blacklist and additional tab on the left sidebar

**Comment by Finch290** [comment]
i'm using jellyseer for quite some times now and it's been amazing, thank you for that! being able to filter and exclude already watched movies and tv shows would be a great improvement. i like to visit the trending page or every studio or service page but i loose a lot of times looking for what i never watched trying to figuring out what it is.
